xarray/tests/test_coding_times.py

```diff
@@ -220,7 +220,7 @@ def test_decode_standard_calendar_inside_timestamp_range(
 ) -> None:
     import cftime
 
-    units = "days since 0001-01-01"
+    units = "hours since 0001-01-01"
```

Comment:
Using encoding units of "days" unnecessarily led the times to be encoded with floats, which prevented asserting exact equality in this test.  Testing floating point time decoding was not the point of this test, so I changed to encoding the times with units of "hours" instead.